src/main/java/org/testeditor/fixture/web/angularjs/materialdesign/NgMdWebDriverFixture.aml

```diff
@@ -32,53 +32,48 @@ interaction type setActiveBrwoser {
 }
 
 interaction type typeIn {
-	template = "Type in" ${element} "value" ${value}
+	template = "Enter" ${element} "into" ${value}
 	method = NgMdWebDriverFixture.typeInto(element, value)
 }
 
 interaction type sendEnter {
-	template = "press enter in" ${element}
+	template = "Press enter on" ${element}
 	method = NgMdWebDriverFixture.pressEnterOn(element)
 }
 
 interaction type clear {
-	template = "Clear value in" ${element}
+	template = "Clear" ${element}
 	method = NgMdWebDriverFixture.clear(element)
 }
 
 interaction type click {
-	template = "click on" ${element}
+	template = "Click" ${element}
 	method = NgMdWebDriverFixture.clickOn(element)	
 }
 
 interaction type selectInSelection {
-	template = "select " ${value} "in" ${element}
+	template = "Select " ${value} "from" ${element}
 	method = NgMdWebDriverFixture.selectElementInSelection(element,value)	
 }
 
 interaction type getAvailableOptions {
-	template = "available Options in" ${element}
+	template = "Available options in" ${element}
 	method = NgMdWebDriverFixture.getOptionsInSelection(element)	
 }
 
 interaction type readValue {
-	template = "read value from" ${element}
+	template = "Read" ${element}
 	method = NgMdWebDriverFixture.readValue(element)	
 }
 
-interaction type selectCurrentDate {
```

Comment:
Is this not needed anymore? I remember a test of Karsten where a date from the date picker needed to be selected ...